src/frame.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+class Frame {
+
+  static ERROR_FRAME_NUMBER_LESS_THAN_1 = `Frame number can't be less than 1`;
+  static ERROR_FRAME_NUMBER_MORE_THAN_10 = `Frame number can't be more than 10`;
+
+  constructor(number) {
+    if (number < 1) {
+      throw new Error(Frame.ERROR_FRAME_NUMBER_LESS_THAN_1);
```

Comment:
I like this use of static above, also your code is very readable and easy to follow which is great